api/routes.go

```diff
@@ -74,5 +74,5 @@ func RegisterRoutes() {
 	shared.AddRoute("/api/metadata", "api-metadata", shared.WrapPermissiveCORS(apiMetadataHandler))
 
 	// API endpoint for modifying Metadata.
-	shared.AddRoute("/api/metadata/triage", "api-metadata-triage", shared.WrapTrustedCORS(apiMetadataTriageHandler, []string{"https://jgraham.github.io"}, []string{"PATCH"}))
+	shared.AddRoute("/api/metadata/triage", "api-metadata-triage", shared.WrapTrustedCORS(apiMetadataTriageHandler, shared.CORSList, []string{"PATCH"}))
```

Comment:
@Hexcles  Looks like `/api/metadata/triage` is not returning JSON here and this API returns a PR URL in text (as w.write() detects the return type automatically). If I wrap shared.WrapApplicationJSON around it, it would possibly break jgraham's code?

Reply:
We shouldn't wrap a plaintext response with `shared.WrapApplicationJSON`, but sorry I don't quite see your point here?

Reply:
Sorry, /login/status only returns plaintext in my original implementation until your [comment](https://github.com/web-platform-tests/wpt.fyi/pull/1826#discussion_r386674858), and I thought you suggested that all APIs should return in JSON format regardless.